sdk/docs/BUILD.bazel

```diff
@@ -15,14 +15,6 @@ load(
     "lf_version_configuration",
 )
 
-exports_files(
```

Comment:
Am I reading it right, you are getting rid of the export example?

Reply:
Yes. It was not referenced and we removed the command already ages ago from the daml assistant because otherwise it would have to be updated to ledger api v2.